### DISCUSSION TITLE
installing sudo

### DISCUSSION THREAD

**Discussion by battlegamer21** [nemanja-m/gaps#39] [Q&A] [answered]
hey I am running the gaps jigsaw algorithm but I am unable to run **sudo apt-get install python-tk** on VS Studio on my windows pc how  am I able to download it 

P.S It says sudo commandnotfound exception @nemanja-m

**Comment by nemanja-m** [comment, marked as answer]
👋 Hi @battlegamer21 

`sudo apt-get install python-tk` is intended to work on Linux-based systems with APT so the command is not supposed to work on Windows. One way to run this command on Windows is via [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install).

Hopefully, `tkinter` is included in the Python 3 core. If you use Python 3, you don't need to install `tkinter` separately.
